libs/barista-components/slider/README.md

```diff
@@ -0,0 +1,62 @@
+# Slider
+
+`DtSlider` is a component that mimics the native html range element. It also
+contains an input filed, that is used to display the current value of the slider
+itself, as well as providing as input for the slider. The component can have two
+additional components in itself as labels. The first label is for the whole
+slider component and the second is for the unit of the slider value.
+
+<ba-live-example name="DtExampleSimpleSlider"></ba-live-example>
+
+## Inputs
+
+| Name       | Type      | Default | Description                                             |
+| ---------- | --------- | ------- | ------------------------------------------------------- |
+| `value`    | `number`  | `0`     | The initial value for the slider                        |
+| `disabled` | `boolean` | `false` | Whether the slider is disabled.                         |
+| `min`      | `number`  | `0`     | The minimum value the slider can take.                  |
+| `max`      | `number`  | `10`    | The maximum value the slider can take.                  |
+| `step`     | `number`  | `1`     | The value of the minimum increment and decrement value. |
+
```

Comment:
Please add the outputs to the README as well. As they are part of the public API for the consumer.

Reply:
I'll add it, and I realized I missed the emit of the output event emitter so thanks :)